textpipe/operation.py

```diff
@@ -15,7 +15,7 @@ class Operation:
     Base class for pipeline operations.
```

Comment:
As operations are renamed to steps: rename this to `steps.py`?

Reply:
Each pipeline `step` is made out of an available `operation` In theory you could implement steps that can execute the same `operation` with different arguments
Thus I think the separation of steps/operations is still valid

Reply:
Right, I have the feeling that clarifying this distinction could be good, e.g., in the README or docstring? It originally confused me too, but I'm not sure if it's because I reason from the pipeline/operations of before, or whether it is because operations and steps are in each other's space.

Reply:
(I mean clarifying/making explicit the `steps` / `operations` distinction, and I think @bcambel's sentence _'Each pipeline `step` is made out of an available `operation`. In theory you could implement steps that can execute the same operation with different arguments'_ pretty much covers it 🤓 )

Reply:
@graus good idea!

Reply:
But maybe not here, but in #100 ?